services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -614,7 +625,8 @@ private boolean createOrVerifyInternalStore(
       String storeDescriptor,
       String keySchema,
       String valueSchema,
-      int partitionCount) {
+      int partitionCount,
+      Supplier<UpdateStoreQueryParams> updateStoreQueryParamsSupplier) {
```

Comment:
Instead of the `Supplier`, can't we just provide the `UpdateStoreQueryParams` object?